libraries/AP_Motors/AP_MotorsTri.h

```diff
@@ -59,6 +65,7 @@ class AP_MotorsTri : public AP_MotorsMulticopter {
     void                thrust_compensation(void) override;
 
     const char* _get_frame_string() const override { return "TRI"; }
+    const char*  get_type_string() const override { return _pitch_reversed ? "pitch-reversed" : "default"; }
```

Comment:
I would be tempted to just leave it as `""` for the none reversed case, just so there is no change.

Reply:
No change would be to have it report "default", which we don't want, right? I could change it in the json manually I suppose, but it feels like this change should have gotten introduced in the original `_pitch_reversed` PR.

Reply:
I could also make a specific handling of the reverse naming in the motor test source instead of the mixer, but I'd really rather do something here.

Reply:
Doesn't it currently just report as "TRI"? This change would make it "TRI/default"?

Reply:
Oh, I forgot that adding the "default" was part of this PR. I had no memory of that🤦

Gotcha. I'll replace "default" with "", but keep the "pitch-reversed" then?

Reply:
Yeah, sounds good